Makefile

```diff
@@ -0,0 +1,15 @@
+all: test
```

Comment:
Also, i'm not advocating the horrendous autotools or Cmake makefiles that are so confusing and hard to figure out. I'm advocating basic things like this:
- https://github.com/jbenet/macos-ssh-proxy/blob/master/Makefile
- https://github.com/jbenet/go-random-files/blob/master/Makefile
- https://github.com/jbenet/gob-js-test/blob/master/Makefile (particularly see the deps part)

obviously none of this is perfect, and i dont use them in all my repos, but i've learned that it's a great way to signal "what this thing should do" and "how to use this thing" in a language everyone can understand.